apps/jackbnb-fe/src/components/PrimaryButton/index.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { Button } from "@chakra-ui/react";
+
+function PrimaryButton() {
+  return (
+    <Button bg="#58DE15" w="300px" h="50px">
+      Boton Primario
```

Comment:
It should use children

Reply:
```suggestion
      {children}
```

note: the children come from the component props
```
const PrimaryButton: FC = ({children}) => {
```